wls-gui-admintool/tests/common/CommonTestDataFactory.ts

```diff
@@ -0,0 +1,18 @@
+export function useCommonTestDataFactory() {
+  function generateNumberRandom(length: number): number {
```

Comment:
```suggestion
  function generateRandomNumber(length: number): number {
```
liest sich einfacher

Reply:
fixed mit 750ebde9cbfeba749c57d421fbb41f02d49a390d